pyproject.toml

```
@@ -23,22 +24,29 @@ classifiers = [
 
 requires-python = ">=3.8"
 dependencies = [
-    "numpy >= 1.8"
+    "numpy >= 1.8",
+    "fortranformat",
 ]
 
 [project.optional-dependencies]
 tests = [
     "pytest >= 3.3.0",
     "pytest-cov",
 ]
+docs = [
+    "sphinx == 3.5.4",
+    "myst_parser == 0.14.0",
+    "jinja2 < 3.1.0",
+    "sphinx_rtd_theme",
```

I have a feeling these are just copy-pasted from pyrokinetics :)

The sphinx version is very old and doesn't work with python >= 3.10, and I think myst isn't needed for this project. These two should be changed at least.

This is personal preference, but I think `sphinx-book-theme` looks nicer than the RTD theme.

A couple of suggestions for other useful plugins:

```suggestion
     "sphinx >= 5.3",
     "sphinx-book-theme >= 0.4.0rc1",
     "sphinx-argparse-cli >= 1.10.0",
     "sphinx_autodoc_typehints >= 1.19",
```